src/main/java/seedu/address/logic/commands/schedule/EditScheduleCommand.java

```diff
@@ -22,55 +24,71 @@
 /**
  * Edits the details of an existing Client in the address book.
  */
-public class RescheduleCommand extends Command {
+public class EditScheduleCommand extends Command {
 
-    public static final String COMMAND_WORD = "reschedule";
+    public static final String COMMAND_WORD = "editschedule";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Reschedule a client with another session. "
-            + "Parameters: INDEX (must be a positive integer)\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": schedules a client with another session. \n"
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: "
+            + PREFIX_CLIENT_INDEX + "CLIENT "
             + PREFIX_SESSION_INDEX + "SESSION "
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_SESSION_INDEX + "1 ";
+            + PREFIX_UPDATED_SESSION_INDEX + "UPDATED SESSION "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_CLIENT_INDEX + "1 "
+            + PREFIX_SESSION_INDEX + "1 "
+            + PREFIX_UPDATED_SESSION_INDEX + "1 ";
```

Comment:
should this be inside [] to show this is optional?

Reply:
oo okay sure i'll make that change, didn't know that it signifies that